tests/libcxx/usual_matrix.lst

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-PM_CL="/nologo /Od /W4 /w14061 /w14242 /w14265 /w14582 /w14583 /w14587 /w14588 /wd4643 /w14749 /w14841 /w14842 /w15038 /w15214 /w15215 /w15216 /w15217 /sdl /WX /Zc:strictStrings /D_ENABLE_STL_INTERNAL_CHECK /bigobj /FImsvc_stdlib_force_include.h /permissive-"
+RUNALL_INCLUDE ..\universal_prefix.lst
+RUNALL_CROSSLIST
+PM_CL="/wd4643 /FImsvc_stdlib_force_include.h /permissive-"
 RUNALL_CROSSLIST
 PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /analyze"
 PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing /EHsc /MTd /std:c++latest"

```

Comment:
(1) There's quite a bit of overlap in `PM_CL` in the third section that could be extracted into the second, and (2) I find it peculiar that we explicitly specify `_ITERATOR_DEBUG_LEVEL` only for the cl config:
```suggestion
PM_CL="/wd4643 /FImsvc_stdlib_force_include.h /permissive- /EHsc /MTd /std:c++latest /D_ITERATOR_DEBUG_LEVEL=2"
RUNALL_CROSSLIST
PM_CL="/analyze"
PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing"
```

Reply:
Agreed - the explicit IDL=2 was a relic of being derived from the `std` configurations, but it's unnecessary here.

Reply:
Done - I dropped the unnecessary IDL=2 and rearranged the options slightly.